ballerina/tests/client_endpoint_test.bal

```diff
@@ -460,7 +460,7 @@ public function testRenameDirectory() {
     string newName = "/home/in/test";
     Error? response1 = clientEp->rename(existingName, newName);
     if (response1 is Error) {
-        log:printError("Error in renaming directory", 'error = response1);
+        test:assertFail(msg = "Error while invoking `rename` operation");
```

Comment:
Since parenthesis are optional in ballerina better to ommit them in control structures (if-else, while etc.).

Reply:
Removed unnecessary parentheses found in the whole module.